github/commitstatus.go

```diff
@@ -31,12 +31,14 @@ func (e *StatusError) Error() string {
 	return fmt.Sprintf("%s\n%s", e.What, e.Details)
 }
 
+// Default GitHub hostname
```

Comment:
// GhDefaultHostname is the default GitHub hostname (used for git but not for the API)

Reply:
We begin the comment with the same name as the variable because this is the hint for the Go doc system to associate the comment to the symbol.

Reply:
Fixed in: https://github.com/Pix4D/cogito/pull/144/commits/66f1cf52187b4780f4fb0775009bab43f2a2155d